src/main/java/mate/academy/springboot/swagger/util/SortOrderUtil.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SortOrderUtil {
+    public Sort sortBy(String sortBy) {
+        List<Sort.Order> orders = new ArrayList<>();
+        if (sortBy.contains(":")) {
+            String[] sortingFields = sortBy.split(";");
+            for (String field: sortingFields) {
+                Sort.Order order;
+                if (field.contains(":")) {
+                    String[] fieldsAndDirections = field.split(":");
+                    order = new Sort.Order(Sort.Direction.valueOf(fieldsAndDirections[1]),
+                            fieldsAndDirections[0]);
+                } else {
+                    order = new Sort.Order(Sort.Direction.DESC, field);
+                }
+                orders.add(order);
+            }
+        } else {
+            Sort.Order order = new Sort.Order(Sort.Direction.DESC, sortBy);
+            orders.add(order);
+        }

```

Comment:
Will it work with "...?sortBy=price;title"?
And I think default direction ASC will be better.
You can fix it, but this is my advice)